src/js/es5.js

```diff
@@ -2,7 +2,7 @@ import './config/staging.js';
 import '@webcomponents/webcomponentsjs';
 import 'whatwg-fetch';
 import './polyfills/endswith.js';
-import '@cagov/accordion';
+// import '@cagov/accordion'; accordion indicator svgs are broken, disable accordions in IE until fixed
```

Comment:
commenting out accordion stuff in the IE only js file because the svg open/close indicator isn't working in IE.

right now in production all accordions are off due to the js errors so this does not change the state of production, we should fix the svgs and turn accordions back on for IE though

when accordions are not working they show up fully expanded so all content is still readable